### ISSUE TITLE
Unity Fractal Server issue

### ISSUE THREAD

**Issue by parandham03** [CatalystCode/3dtoolkit#169]
Hi,
  I am running signalling server locally. When running Unity Fractal Server there is nothing in drop down to select. Shall i run StreamingDirectxClient as client and Unity Fractal Server as server? If i run like that there is nothing in unity server drop down. Please help me to figure out this.

![unityserver](https://user-images.githubusercontent.com/38241337/38596563-dfba8026-3d6f-11e8-800d-3cb04599da28.png)

**Comment by parandham03** [comment]
Hi,
  It works now. Based the thread [https://github.com/CatalystCode/3dtoolkit/issues/135](url) i have deleted Authentication failed in webrtcConfig. No its fine. But when connecting to a directx client i can see 3d cubes rendering in directx client. But in Unity server the UI is still there. When i connect unity server with unity client same thing happening in server. And there is no cubes showing in unity client.

**Comment by phongcao** [comment]
Looks like there is something wrong with the Unity FractalStream server. Can you enable "Development build" so that we can see if there is any error? My guess is there is something wrong with the dll loading.

**Comment by parandham03** [comment]
Is it possible to deploy Unity Fractal Server to hololens and unity client in desktop?

**Comment by phongcao** [comment]
No, it's not supported since Unity FractalStream server uses Win32 APIs.

**Comment by treisto** [comment]
@parandham03 Hello, did you figure out why the Unity Fractal Server's initial connection UI was not disappearing after connecting to client? I have the same issue. 

Also, on Hololens Unity client, I get short messages when signing-in to signalling server and after connecting to the streaming server, which each disappear with fade; but afterwards, I have only a black screen on Hololens Unity client.

If I connect the server to a win32 client, I get the spinning cubes on the client, but the UI is still there on the Unity Fractal Streaming server.

**Comment by treisto** [comment]
Hi phongcao, thanks a lot for the answer. Indeed, I am initialising the connection from the server.
I have tried the Dx Holo client, and everything looks the same. I have also tried some other combinations of servers and clients, but until now the only one that somehow works is Unity Fractal Server + DxWin32 client - I have the fractals on the client, but the server screen is still UI on black background. 

To be more clear and specific, I have opened issue #187 where I have attached logs for server + client. The weird log on the signalling server seems to appear only when the WI-FI router used for comm and signalling is not connected to the internet - I presume it is not mandatory to connect to internet if I am running on local Wi-FI ?
